src/theme/theme.ts

```diff
@@ -83,7 +83,7 @@ const getGlobalStyles = async () => {
       --divider-color: ${light.dividerColor};
       --secondary-color: ${light.secondaryColor};
       --background-color-dark: ${light.backgroundColorDark};
-      --background-color: ${light.backgroundColor};
+      --background-color: ${light.light};
       --highlight-background: ${light.highlightBackground};
```

Comment:
Revert this change. While `light` and `backgroundColor` is currently the same, this creates a linkage that could cause issues down the line. Changing the `light` color would affect the background in multiple places, which may not be what we desire.